src/clients/inst_store_upgrade_software.rb

```diff
@@ -108,8 +108,8 @@ def transactional_packages(status)
       names = Pkg.GetPackages(status, names_only)
 
       names.select do |name|
-        packages = Pkg.ResolvableProperties(name, :package, "")
-        packages.any?{ |p| p["transact_by"] == :user || p["transact_by"] == :app_high }
+        Pkg.PkgPropertiesAll(name).any? { |p| (p["transact_by"] == :user || p["transact_by"] == :app_high) &&
```

Comment:
Style: use `do/end` style for multiline blocks and put the condition on a separate line. Rubocop would complain here if we enable it in the future. :wink: